src/patternfly/themes/dark/scss-variables.scss

```diff
@@ -1,3 +1,6 @@
+@import "../../sass-utilities/functions"; // functions
+@import "colors"; // dark colors
```

Comment:
I might put these in the main dark stylesheet here - https://github.com/patternfly/patternfly/blob/cc6c2257ad05e4e7b5a086bf8ca2d8e3beb993ef/src/patternfly/themes/dark/_patternfly-theme-dark.scss#L1-L9

And mimic the ordering of the existing sass-utilities import order from https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/_all.scss. 

It looks like [colors.scss is already imported](https://github.com/patternfly/patternfly/blob/cc6c2257ad05e4e7b5a086bf8ca2d8e3beb993ef/src/patternfly/themes/dark/_patternfly-theme-dark.scss#L4), so I wouldn't think it needs to be imported again?

Reply:
@mcoker updated!